pallets/ddc-staking/src/lib.rs

```diff
@@ -507,35 +618,28 @@ pub mod pallet {
 			Ok(())
 		}
 
-		/// Update the DDC staking configurations .
+		/// Set custom DDC staking settings for a particular cluster.
 		///
-		/// * `storage_bond_size`: The active bond needed to be a Storage node.
-		/// * `edge_bond_size`: The active bond needed to be an Edge node.
+		/// * `settings` - The new settings for the cluster. If `None`, the settings will be removed
+		///   from the storage and default settings will be used.
 		///
 		/// RuntimeOrigin must be Root to call this function.
 		///
-		/// NOTE: Existing nominators and validators will not be affected by this update.
+		/// NOTE: Existing CDN and storage network participants will not be affected by this
+		/// settings update.
 		#[pallet::weight(10_000)]
-		pub fn set_staking_configs(
+		pub fn set_settings(
 			origin: OriginFor<T>,
-			storage_bond_size: ConfigOp<BalanceOf<T>>,
-			edge_bond_size: ConfigOp<BalanceOf<T>>,
+			cluster: ClusterId,
+			settings: Option<ClusterSettings<T>>,
 		) -> DispatchResult {
 			ensure_root(origin)?;
```

Comment:
This should go through governance, right?

Reply:
Yes, `ensure_root` allows root origins like `sudo` or `pallet-democracy` proposals.